routing_common/transit_serdes.hpp

```diff
@@ -87,10 +95,11 @@ class Deserializer
     ReadPrimitiveFromSource(m_source, t);
   }
 
-  template<typename T>
-  typename std::enable_if<std::is_floating_point<T>::value>::type operator()(T & t, char const * name = nullptr)
+  void operator()(double & d, char const * name = nullptr)
   {
-    NOTIMPLEMENTED();
+    uint32_t ui;
+    (*this)(ui, name);
+    d = Uint32ToDouble(ui, kMinDoubleAtTransit, kMaxDoubleAtTransit, POINT_COORD_BITS);
```

Comment:
So, you encode weight (seconds), and use POINT_COORD_BITS for that? Looks not good to me.

Reply:
Yes. If I understand correctly the meaning of |coordBits| param it should be 31 in this case. Right?